packages/material-ui/src/ButtonBase/TouchRipple.test.js

```diff
@@ -44,20 +43,16 @@ describe('<TouchRipple />', () => {
     };
   }
 
-  before(() => {
-    classes = getClasses(<TouchRipple />);
-  });
-
-  describeConformance(<TouchRipple />, () => ({
-    classes,
+  describeConformanceV5(<TouchRipple />, () => ({
+    classes: {},
     inheritComponent: 'span',
     mount,
     refInstanceof: Object,
-    skip: ['componentProp', 'refForwarding'],
+    skip: ['componentsProp', 'refForwarding', 'themeComponents'],
```

Comment:
It's private component, we don't need to test that `theme.components` works as expected.

Reply:
We should probably encode that instead of skipping individual tests. That way we explicitly test whether a component is considered private or public. I'll write it down as a follow-up.

Generally, I don't think internal components should be part of `describeConformance`. But that's for another time.